app/src/main/res/values/strings.xml

```diff
@@ -98,6 +98,7 @@
   <string name="err_invalid_phone">Please enter a valid phone number</string>
   <string name="err_address_blank">Please enter an address</string>
   <string name="err_cant_load_photo">Unable to load photo</string>
+  <string name="err_cant_load_photo_bad_extension">Unable to load photo. File extension is not accepted.</string>
```

Comment:
Can you make this error message a bit more user friendly?  They don't know nor care about file extensions.  Maybe just "Sorry, FieldTheBern could not load your image"